versioned_docs/version-24.04/integrations/stream-connectors.md

```diff
@@ -9,6 +9,25 @@ Stream connectors transfer in real time the flow of monitoring data coming from
 
 ## 2024
 
+### August
+
+<Tabs groupId="sync">
+<TabItem value="Fix" label="Fix">
+
+- [**Splunk Metrics**](data-analytics/sc-splunk-metrics.md) - Fixed an issue with **max_buffer_size** parameter in Splunk stream connector.
+- [**Clickhouse**](data-analytics/sc-clickhouse.md) - Added a protection on metric names if it contains spaces.
+- [**Influxdb2 Metrics**](data-analytics/sc-influxdb2-metrics.md) - Added a protection on metric names if it contains spaces.
+- [**Centreon library: sc_macro**](https://github.com/centreon/centreon-stream-connector-scripts/blob/develop/modules/centreon-stream-connectors-lib/sc_macro.lua) - Fixed an issue with **_scshort flag** leading to macro name of an event instead of its value.
```

Comment:
```suggestion
- [**Centreon library: sc_macro**](https://github.com/centreon/centreon-stream-connector-scripts/blob/develop/modules/centreon-stream-connectors-lib/sc_macro.lua) - Fixed an issue with **_scshort flag** leading to the macro name of an event being displayed instead of its value.
```